grumblr/db/migrate/20150519205332_add_body_author_to_grumble.rb

```diff
@@ -0,0 +1,6 @@
+class AddBodyAuthorToGrumble < ActiveRecord::Migration
+  def change
+    add_column :grumbles, :body, :string
+    add_column :grumbles, :author, :string
+  end
+end
```

Comment:
mehhh I think I just like getting the migration created before adding the
additional info?  Baby steps?

On Wed, May 20, 2015 at 11:07 AM, Andrew Kim <notifications@github.com>
wrote:

> In grumblr/db/migrate/20150519205332_add_body_author_to_grumble.rb
> <https://github.com/ga-dc/grumblr/pull/12#discussion_r30712074>:
>
> > @@ -0,0 +1,6 @@
> > +class AddBodyAuthorToGrumble < ActiveRecord::Migration
> > +  def change
> > +    add_column :grumbles, :body, :string
> > +    add_column :grumbles, :author, :string
> > +  end
> > +end
>
> interesting, i'm curious as to why you added an additional migration to
> add columns instead of just including them in your initial migration you
> created for grumbles.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-dc/grumblr/pull/12/files#r30712074>.
>



-- 
Diana Jurgielewicz
dmjurg@gmail.com
(703) 380-2577